Topic 1/src/exercise1to6/cart/MarketManager.java

```diff
@@ -0,0 +1,31 @@
+package exercise1to6.cart;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Observable;
+import java.util.Observer;
+
+public class MarketManager implements Observer {

```

Comment:
Market Manager must not be modeled as an Object. Notifications are by email. There should be a mailing list that will contain Market Manager's email and the Observer should be a EmailNotifactionService with a mock implementation.